src/test/java/io/ticktok/TicktokTest.java

```diff
@@ -50,8 +50,8 @@ private void verifyCallbackWasntDoneSynchronicity(CountDownLatch countDownLatch)
         assert countDownLatch.getCount() == 1;
     }
 
-    @After
-    public void tearDown() {
+    @AfterAll
+    public static void tearDown() {
```

Comment:
with junit 5 you can create the class once for all test with @TestInstance(PER_CLASS)

Reply:
fixed, good tip